layerInfo.json

```diff
@@ -157,6 +157,17 @@
 					"Server-Shared-Utils"
 				]
 			},
+			"OdspClient": {
+				"dirs": ["packages/service-client/odsp-client"],
+				"deps": [
+					"Container-Definitions",
+					"Framework",
+					"Runtime",
+					"Routerlicious-Driver",
+					"Server-Shared-Utils",
```

Comment:
I'd think there should be no dependencies for the last 3 items, no?

Reply:
AzureClient will still remain for now. But rest can be removed :)